8.0/Dockerfile

```diff
@@ -125,20 +125,8 @@ RUN RUNTIME_DEPS="$(scanelf --needed --nobanner --recursive /usr/local \
 # copy custom.ini settings
 COPY craft-cms.ini /usr/local/etc/php/conf.d/
 
-# copy the blackfire script
-COPY blackfire-install.sh /tmp/blackfire-install.sh
-
-# run the installation script
-RUN set -ex && sh /tmp/blackfire-install.sh && rm -rf /tmp/blackfire*
-
-# make the directories and set permissions
-RUN mkdir -p /app
-
-# set the permissions on the
-RUN chown -R www-data:www-data /app
-
```

Comment:
@jawys is there a reason you dropped the `chown`?

Reply:
see this:
```console
❯ echo "
  FROM php:8.2-fpm-alpine3.18
  # run container as the www-data user
  USER www-data
  # set the working directory for convenience
  WORKDIR /app
  " | docker build - -t test && docker run --rm -it test ls -lad /app
[+] Building 0.6s (6/6) FINISHED                                                          docker:desktop-linux
 => [internal] load build definition from Dockerfile                                                      0.0s
 => => transferring dockerfile: 177B                                                                      0.0s
 => [internal] load .dockerignore                                                                         0.0s
 => => transferring context: 2B                                                                           0.0s
 => [internal] load metadata for docker.io/library/php:8.2-fpm-alpine3.18                                 0.5s
 => [1/2] FROM docker.io/library/php:8.2-fpm-alpine3.18@sha256:adb9e7712d82a8c50fd01b73cd148859c4562bbe8  0.0s
 => CACHED [2/2] WORKDIR /app                                                                             0.0s
 => exporting to image                                                                                    0.0s
 => => exporting layers                                                                                   0.0s
 => => writing image sha256:36352bf42020368e3e118c009f52b17dd7090574a73682f21dbdaf3bae1da351              0.0s
 => => naming to docker.io/library/test                                                                   0.0s
drwxr-xr-x    2 www-data www-data      4096 Aug 29 12:46 /app
```